test_coco.py

```diff
@@ -266,15 +270,59 @@ def test(data,
             cocoEval.accumulate()
             cocoEval.summarize()
             map, map50 = cocoEval.stats[:2]  # update results (mAP@0.5:0.95, mAP@0.5)
+            
+            if evaluate_fbeta:
+                cocoEval.accumulateFBeta()
+                cocoEval.summarizeFBetaScores(average=fbeta_average_method)
+                f1_score, _, _, _ = cocoEval.getBestFBeta(beta=1, iouThr=iou_thres, average=fbeta_average_method)
+                f2_score, _, _, _ = cocoEval.getBestFBeta(beta=2, iouThr=iou_thres, average=fbeta_average_method)
+                print(f"Results:")
+                s = ('%20s ' * 4) % ('f1-score', 'f2-score', 'map', 'map50')
+                print(f"{s}")
+                s = ('%20s ' * 4) % (f1_score, f2_score, map, map50)
+                print(f"{s}")
```

Comment:
Hmm not sure if this would be confusing, because the values are calculated from different parameters